src/components/synonyms/agSynonymTable.vue

```diff
@@ -138,6 +125,23 @@ export default {
           cellEditor: "agLargeTextCellEditor"
         }
       ];
+      if (this.editable) {
+        // Add the save button
+        colDefs.push({
+          flex: 0,
+          width: 75,
```

Comment:
```suggestion
          width: 85,
```
this will make the cell wide enough to remove the ellipsis that I am seeing in the cell to the right of the button

Reply:
I've made some changes on the ticket following this that may invalidate these changes but this certainly makes sense here.  To be honest, I only tried 50px - 75px - 100px.